karma.sauce.conf.js

```diff
@@ -23,6 +23,12 @@ const customLaunchers = {
 		base: 'SauceLabs',
 		browserName: 'microsoftedge',
 		platform: 'Windows 10'
+	},
+	edgeLegacy: {
+		base: 'SauceLabs',
+		browserName: 'microsoftedge',
+		browserVersion: '18.17763',
```

Comment:
Not sure what I'm doing wrong here but sauce seems unwilling to run a version of edge that isn't 80